packages/expo-dev-launcher/ios/EXDevLauncherURLHelper.swift

```diff
@@ -65,11 +65,13 @@ public class EXDevLauncherURLHelper: NSObject {
 
   @objc
   public static func replaceEXPScheme(_ url: URL, to scheme: String) -> URL {
-    var components = URLComponents.init(url: url, resolvingAgainstBaseURL: false)!
-    if components.scheme == "exp" {
-      components.scheme = scheme
+    if var components = URLComponents(url: url, resolvingAgainstBaseURL: false) {
+      if components.scheme == "exp" {
+        components.scheme = scheme
+      }
+      return components.url!
```

Comment:
```suggestion
      return components.url ?? ""
```

Reply:
Updated the whole function so we no longer force unwrap and just default to the input url

Reply:
Actually this is a URL not a String, sorry I read the code too quickly. Let me have another look

Reply:
Your changes look good now, approved